src/integrationTest/java/uk/gov/hmcts/reform/jps/controllers/RecordSittingRecordsControllerITest.java

```diff
@@ -40,19 +52,36 @@ public class RecordSittingRecordsControllerITest {
     @Autowired
     private ObjectMapper objectMapper;
 
-    @Test
+    @MockBean
+    private SecurityUtils securityUtils;
+
+    @MockBean
+    private UserInfo userInfo;
+
+
+    @ParameterizedTest
+    @CsvSource({"recordSittingRecordsReplaceDuplicate.json,200,4918178",
+        "recordSittingRecords.json,201,4918500"})
+    @Sql(scripts = {DELETE_SITTING_RECORD_STATUS_HISTORY, ADD_SITTING_RECORD_STATUS_HISTORY})
     @WithMockUser(authorities = {"jps-recorder", "jps-submitter"})
```

Comment:
can we start adding the roles in a constant class to avoid typos please

Reply:
Added a constant interface